packages/commonwealth/client/scripts/views/pages/create_community/chain_input_rows.tsx

```diff
@@ -14,6 +15,23 @@ import type {
   UseEthChainFormFieldsHookType,
 } from './types';
 
+export async function updateAdminRole(chainId: string) {
```

Comment:
We may want a better name for this function, "updateAdminRole", which seems to completely re-initialize the chain. Alternatively, why not move the roles re-query into initAppState?